cirq/sim/act_on_args.py

```diff
@@ -26,9 +26,9 @@ class ActOnArgs:
 
     def __init__(
         self,
-        axes: Iterable[int],
         prng: np.random.RandomState,
-        log_of_measurement_results: Dict[str, Any],
+        axes: Iterable[int] = None,
+        log_of_measurement_results: Dict[str, Any] = None,
```

Comment:
[No change needed] IIUC, this class was added after the v0.10 release so changing its argument ordering like this is safe.